### DISCUSSION TITLE
Is there a way to get phrase search work with Orama?

### DISCUSSION THREAD

**Discussion by litcodeur** [oramasearch/orama#729] [Q&A] [answered]
I'm trying to see if it is possible to do a phrase search. For ex: If I search for "a string in an order", I would like it so that the contents within the double quotes match exactly and in the order they were given, so it wouldn't match for just "a","string", "in".... but the whole string and in the order.

For some additional reference here's meilisearch's docs on phrase search: https://www.meilisearch.com/docs/reference/api/search#phrase-search. 

Thanks!

**Comment by micheleriva** [comment, marked as answer]
Hi @litcodeur, this is not currently possible. We have a PR almost ready to enable this, but it may take a bit longer for us to merge it - as it forces us to refactor a bunch of things.

**Reply by litcodeur** [reply]
Thanks for the update @micheleriva. Good to know it's in progress. Lmk if there's anything I might be able to contribute to, to help speed it up.

**Reply by jackkoppa** [reply]
Thanks so much for the response here, @micheleriva 🙇 

I _doubt_ I'd be able to help a ton, but was there any WIP PR on this that I could perhaps take a look at? Would love to enable exact phrase matching as an option.

**Reply by isaachinman** [reply]
@micheleriva Any update on this?

**Reply by micheleriva** [reply]
sadly no progress right now, we focused more on [OramaCore](https://github.com/oramasearch/oramacore). I'd love to get back to this as soon as I have more time
